README.md

```diff
@@ -25,8 +25,8 @@ Material design: [Dropdown buttons](https://www.google.com/design/spec/component
 another element that represents "content", to create a dropdown menu that
 displays the "content" when the "trigger" is clicked.
 
-The child element with the class `dropdown-trigger` will be used as the
-"trigger" element. The child element with the class `dropdown-content` will be
+The child element with the slot `dropdown-trigger` will be used as the
```

Comment:
Same thing as iron-dropdown: this might be clearer changing 'with the slot `dropdown-trigger`' to 'assigned to the `dropdown-trigger` slot' here and on the line below.